src/System.Management.Automation/FormatAndOutput/DefaultFormatters/PowerShellCore_format_ps1xml.cs

```diff
@@ -1103,7 +1103,7 @@ private static IEnumerable<FormatViewDefinition> ViewsOf_System_Management_Autom
                                                 $line = $line.Insert($offsetInLine + $offsetLength, $resetColor).Insert($offsetInLine, $accentColor)
                                             }
 
-                                            $posmsg += ""${accentColor}${lineWhitespace}${ScriptLineNumber} ${verticalBar} ${resetcolor}${line}`n""
+                                            $posmsg += ""${accentColor}${lineWhitespace}${ScriptLineNumber} ${verticalBar} ${resetcolor}${line}""
```

Comment:
I see `$line = $myinv.Line` always ends with new line but I am not sure about ` $line = $_.TargetObject.LineText.Trim()` - can you confirm? Or maybe explicitly check and add newline if absent?

Reply:
That's a good catch.  I'll add a check to add newline if absent.